enterprise/frontend/src/embedding/data-picker/DataSelector.jsx

```diff
@@ -441,20 +379,6 @@ export class UnconnectedDataSelector extends Component {
     return this.props.loading;
   };
 
-  getCardType() {
```

Comment:
First, I removed `selectedDataBucketId === DATA_BUCKET.METRICS` case, then found out that the `default` option is never called.

Basically, `getCardType` is only called inside `SavedEntityPicker` which is only rendered when we either select the models bucket or saved questions bucket. But the saved questions bucket has been removed since the implementation of the simple data picker, so we don't have that option anymore. The only value is now `"model"` so I inline it instead.